databricks/koalas/plot.py

```diff
@@ -508,6 +508,17 @@ def _make_plot(self):
         super(KoalasBarhPlot, self)._make_plot()
 
 
+class KoalasKdePlot(KdePlot, TopNPlot):
+    max_rows = 1000
+
+    def __init__(self, data, **kwargs):
+        super(KoalasKdePlot, self).__init__(self.get_top_n(data), **kwargs)
```

Comment:
From what I know, we should implement the calculation logic (see histogram for instance) via Spark DataFrame unlike other simple charts.

Reply:
oh, apparently it seems it's okay to directly use, i took a double check on Pandas, it's a common kind there unlike boxplot. i tried different cases a bit and they are all good

Reply:
I didn;t take a look closely - it was just my impression. I will double check tomorrow.

Reply:
for big data, calculation for `gaussian_kde` is not correct. I commented it below already. i think a calculation for this needs to be implemented. @HyukjinKwon